### ISSUE TITLE
Expose markdown-it "break" option

### ISSUE THREAD

**Issue by fabiospampinato** [notable/notable#1235]
In order to better support people who break their lines at arbitrary lengths but don't want a newline to be inserted in that case.

**Comment by morciej** [comment]
Yes please! I just converted/edited a large `man` file, and it has line breaks where I don't need them. I expected to see standard Markdown behaviour "**paragraph = lines together**". I would be fine with "**trailing backslash**" to force breaks, but the only option now is "long-long lines" as far as I can tell.
